src/Storefront/Resources/views/storefront/layout/navbar/navbar.html.twig

```diff
@@ -33,12 +33,23 @@
 
                             {% block layout_navbar_menu_item %}
                                 {% if category.type == 'folder' %}
-                                    <li class="nav-item main-navigation-link nav-item-{{ id }} nav-link root nav-item-{{ id }}-link"
+                                    <li class="nav-item main-navigation-link nav-item-{{ id }} nav-link root nav-item-{{ id }}-link{% if hasChildren %} dropdown-toggle{% endif %}"
+                                        data-bs-toggle="dropdown"
```

Comment:
I think `data-bs-toggle="dropdown"` should also be inside `{% if hasChildren %}`